coderefs/coderefs.go

```diff
@@ -15,14 +15,19 @@ import (
 )
 
 func Run(opts options.Options) {
+	if len(opts.ProjKey) > 0 {
```

Comment:
We set this inside run because all instances(github actions, bitbucket, CLI) call to `Run`. Converts legacy `projKey` into a `Project`.